packages/bpk-component-close-button/src/BpkCloseButton.js

```diff
@@ -51,10 +52,12 @@ BpkCloseButton.propTypes = {
   label: PropTypes.string.isRequired,
   onClick: PropTypes.func.isRequired,
   className: PropTypes.string,
+  customIcon: PropTypes.func,
```

Comment:
I've added a new prop to the close button component: `customIcon`. I don't want to document it yet. 

My thinking is that we have a requirement for icon only buttons (close buttons, hamburger buttons etc) that i'd like to make a generic component for. The current `bpk-component-close-button` actually fits these needs, it's just named too specifically. Going forward we can create a `bpk-component-icon-button` which supercedes `bpk-component-close-button` (the underlying implementation of which can just depend on the new icon button component with a close icon hard coded.) Thoughts?

Reply:
We agreed to add a snapshot test for this.